src/main/resources/org/terasology/launcher/views/log-view.fxml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Tab?>
+<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.control.ScrollPane?>
+<?import javafx.scene.control.TextArea?>
+<Tab xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/2.2"
+     fx:controller="org.terasology.launcher.gui.javafx.LogViewController" closable="false" text="Log">
+    <content>
+        <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="336.0" prefWidth="800.0">
+            <children>
+                <ScrollPane
+                        fitToHeight="true" fitToWidth="true"
+                        prefHeight="336.0" prefViewportHeight="336.0" prefViewportWidth="800.0" prefWidth="800.0"
+                        AnchorPane.bottomAnchor="8.0" AnchorPane.leftAnchor="8.0" AnchorPane.rightAnchor="8.0" AnchorPane.topAnchor="8.0">
+                    <content>
+                        <TextArea fx:id="logArea" prefHeight="336.0" prefWidth="200.0" />
+                    </content>
+                </ScrollPane>
+            </children>
+        </AnchorPane>
+    </content>
+</Tab>
```

Comment:
Using a `Tab` as the root level element works perfectly fine for our UI :+1:  but this approach kinda limits our `LogViewController` to be functional only inside a `TabPane`. It should be more appropriate to slightly change our setup like this:
- Lifting the `Tab` part back to `application.fxml` and using the root for `log-view.fxml` as its `content`
- We can simplify the `log-view.fxml` to be just a `AnchorPane`+`TextArea`. There shouldn't be a need for intermediate  `ScrollPane` for now (`TextArea` automatically provides scrollbars when text goes out of bounds)
- Text wrapping should make it easier to see longer messages (but that's quite subjective)

Reply:
Good point! The described change would also make it easier to implement the custom `LogView` control.